third_party/txt/BUILD.gn

```diff
@@ -125,6 +125,12 @@ source_set("txt") {
   deps = [
     "//third_party/skia:effects",
   ]
+
+  if (is_fuchsia) {
```

Comment:
Lets put this in the content handler instead. That way, we can move it out of the engine in one shot instead of having to later patch txt too.